Assets/HoloToolkit/Utilities/Scripts/Editor/InputManagerAxesWindow.cs

```diff
@@ -0,0 +1,295 @@
+            new InputManagerAxis() { name = "CONTROLLER_LEFT_STICK_VERTICAL", dead = 0.19f, sensitivity = 1, invert = true, type = AxisType.JoystickAxis, axis = 2 },
+            new InputManagerAxis() { name = "CONTROLLER_LEFT_STICK_CLICK", positiveButton = "joystick button 8", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "CONTROLLER_RIGHT_STICK_HORIZONTAL", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 4 },
+            new InputManagerAxis() { name = "CONTROLLER_RIGHT_STICK_VERTICAL", dead = 0.19f, sensitivity = 1, invert = true, type = AxisType.JoystickAxis, axis = 5 },
+            new InputManagerAxis() { name = "CONTROLLER_RIGHT_STICK_CLICK", positiveButton = "joystick button 9", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "CONTROLLER_LEFT_TRIGGER", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 9 },
+            new InputManagerAxis() { name = "CONTROLLER_RIGHT_TRIGGER", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 10 },
+            new InputManagerAxis() { name = "XBOX_SHARED_TRIGGER", dead = 0.19f, sensitivity = 1, type = AxisType.JoystickAxis, axis = 3 },
+            new InputManagerAxis() { name = "XBOX_LEFT_BUMPER", positiveButton = "joystick button 4", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_RIGHT_BUMPER", positiveButton = "joystick button 5", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_A", positiveButton = "joystick button 0", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_B", positiveButton = "joystick button 1", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_X", positiveButton = "joystick button 2", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_Y", positiveButton = "joystick button 3", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_VIEW", positiveButton = "joystick button 6", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
+            new InputManagerAxis() { name = "XBOX_MENU", positiveButton = "joystick button 7", gravity = 1000, dead = 0.001f, sensitivity = 1000, type = AxisType.KeyOrMouseButton, axis = 1 },
```

Comment:
Should be `CONTROLLER_MENU` because the Xbox and Motion controller shares this button mapping.

Reply:
True! I didn't change it originally because it only maps to the menu button on the right motion controller. XBOX_VIEW maps to the menu button on the left motion controller. I'm sure there are better names for the two, but I didn't come up with any in the moment haha.

Reply:
Oh, wow, I didn't know that.  Yeah we should really work the names out.

Reply:
We could always add a duplicate entry into the `InputManager.asset` that maps to the same button, but with different names.

Reply:
`CONTROLLER_LEFT_MENU` and `CONTROLLER_RIGHT_MENU` could work, banking on the fact that not many people know the left button on the Xbox controller is actually called View haha...

Reply:
I like it! 😂